### PR TITLE
Chart to deploy sftp inbox based stack

### PR DESCRIPTION
**Pull request by jbygdell**
Includes GH actions for linting and deployment tests.
No integration tests are inclued yet.

### CODE REVIEW

File: sda-svc/values.yaml [hunk 1]
```diff
@@ -0,0 +1,292 @@
+# Default values for localega.
```
Comment (blankdots) [inline sda-svc/values.yaml]:
```suggestion
# Default values for SDA services.
```

**Comment by jbygdell** [comment]
Readme files are still missing, question is if we should create a sda.readthedocs.io as well.

**Comment by blankdots** [comment]
took `neic-sda.readthedocs.io` (pointing to our `neicnordic/localega` fork; the `sda` one was taken, will continue with testing at some point today.

File: sda-svc/values.yaml [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+    nfsPath: ""
+
+
+  broker:
+    host: ""
+    exchange: ""
+    port: 5671
+    verifyHostname: false
+    vhost: "/"
+# This
```
Comment (viklund) [inline sda-svc/values.yaml:101]: What do you mean "This"?

File: sda-svc/values.yaml [hunk 3]
```diff
@@ -0,0 +1,292 @@
+# Default values for SDA services.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+global:
```
Comment (viklund) [inline sda-svc/values.yaml:4]: The global var needs to be documented. I didn't really understand what `external` vs. `internal` actually means for example.

**Pull request merged**